app/views/candidate_mailer/eoc_deadline_reminder.erb

```diff
@@ -2,14 +2,16 @@
 Dear <%= @application_form.first_name %>
 <% end %>
 
-[Submit your application](<%= candidate_magic_link(@application_form.candidate) %>) as soon as you can to get on a course starting in the <%= RecruitmentCycle.current_year %> to <%= RecruitmentCycle.next_year %> academic year.
+[Submit your application for teacher training](<%= candidate_magic_link(@application_form.candidate) %>) as soon as you can to get on a course starting in the <%= RecruitmentCycle.current_year %> to <%= RecruitmentCycle.next_year %> academic year.
 
 <%=  "The deadline to submit your application is #{CycleTimetable.date(:apply_deadline).to_fs(:govuk_time)} on #{CycleTimetable.apply_deadline.to_fs(:govuk_date)} but courses may fill up before then." %>
 
 Courses fill up quickly at this time of year.
 
 # Get help
 
-A teacher training adviser can help you write a strong application:
+If you have questions about your application, you can [get in touch with us on the phone or through live chat](<%= email_link_with_utm_params I18n.t('get_into_teaching.url_online_chat'), 'eoc_deadline_reminder', @application_form.phase %>).
 
-[Get a teacher training adviser](<%= email_link_with_utm_params I18n.t('get_into_teaching.url_get_an_adviser_start'), 'eoc_deadline_reminder', @application_form.phase %>)
+You can also [get a teacher training adviser](<%= email_link_with_utm_params I18n.t('get_into_teaching.url_get_an_adviser_start'), 'eoc_deadline_reminder', @application_form.phase %>) for free, one-to-one support to help you write a strong application.
+
+[Unsubscribe from these messages](<%= candidate_unsubscribe_link(@application_form.candidate) %>) to stop receiving deadline reminders. You will still receive updates about your application.
```

Comment:
Is there anything we can do at the mailer class level to indicate that it's an email that can be unsubscribed to? I can imagine this becoming confusing as to which emails can be unsubscribed from. 
I was thinking like making a `Candidate::NudgeMailer` where this stuff matters and a `Candidate::TransactionMailer` names a vom!

This could be a to do later kind of thought/task

Reply:
I agree -- I think we need to go through out emails and decide which ones are relevant for subscribing / unsubscribing. Beyond the scope of this ticket and it needs some content input as well (eg, if an email is transactional, should we have a message like, ' you are receiving this message because xxx.')